test/components/connect.spec.js

```diff
@@ -12,9 +12,9 @@ describe('React', () => {
         fetchSpies.forEach((spy) => spy())
         return new Promise((resolve) => {
           if (request.url == '/error') {
-            resolve(new window.Response(JSON.stringify({ error: 'e', id: 'not_found' }), { status: 404 }))
+            resolve(new window.Response(JSON.stringify({ error: 'e', id: 'not_found' }), { status: 404, headers: { 'Content-Type': 'application/json' } }))
           } else {
```

Comment:
you could add a `/empty` here instead of going it in the test. this fixture is all kind of messy (my fault), but at this way its consistent